dev/src/reference.ts

```diff
@@ -1725,22 +1775,28 @@ class QueryUtil<
             docs.reverse();
           }
 
-          resolve(
-            query._createSnapshot(
-              readTime,
-              docs.length,
-              () => docs,
-              () => {
-                const changes: Array<
-                  DocumentChange<AppModelType, DbModelType>
-                > = [];
-                for (let i = 0; i < docs.length; ++i) {
-                  changes.push(new DocumentChange('added', docs[i], -1, i));
-                }
-                return changes;
+          if (!output.readTime) {
+            reject(wrapError(Error('No read time'), stack));
+            return;
+          }
```

Comment:
@tom-andersen  see here

Reply:
Comment this out for now, with explanation that 'ExplainOptions' with 'analyze = false` will not have read time.

This is because no results are results are returned in this situation.